### ISSUE TITLE
Modify Permissions Available to the Commanders of a Nation

### ISSUE THREAD

**Issue by connor7000** [mc-ascendancy/ascendancy#151]
From **squid#2303** on Discord:

"Let people other than the _Chancellor_ kick people."

**Comment by connor7000** [comment]
With the current Nation Hierarchy, this idea would mean that both _Commanders_ and the Chancellor would have permissions to run "/nation kick <player>".

Before we address the idea itself, would redesigning the Nation Hierarchy in this way even be practical, @VolticFroogo?

**Comment by VolticFroogo** [comment]
@connor7000, yes this would not be a problem, I could allow commanders to kick only Citizens easily.
